xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml

```diff
@@ -74,6 +73,6 @@
  * @param $imgName The name of the icon to use (without the extension).
```

Comment:
An icon from the XWiki Icon Set.
However, this code is not a part of the Pull Request (I wrote it :p) so I won't charge @Pbast for fixing it :)